src/eba_tips.eliom

```diff
@@ -117,19 +117,20 @@ let%shared block ?(a = []) ~name ~content () =
     if Stringset.mem name seen
     then Lwt.return None
     else begin
-      let close_button = Ot_icons.D.close () in
+      let box_ref = ref None in
+      let close = [%client (fun () ->
+        let () = match !(~%box_ref) with
+          | Some x -> Manip.removeSelf x
+          | None -> () in
+        Lwt.async (fun () -> set_tip_seen ~%name);
+        Lwt.return () : _ -> _) ] in
```

Comment:
Or maybe the question is: why does `close` is wrapped in a `Lwt.async` call? Shouldn't we wait for the action to complete?

i.e. replace 
```
Lwt.async (fun () -> set_tip_seen ~%name) ; Lwt.return () 
```
with
```
set_tip_seen ~%name
```